### DISCUSSION TITLE
strange characters in output

### DISCUSSION THREAD

**Discussion by adam9901** [projectdiscovery/nuclei#2552] [Q&A] [answered]
![nuclei](https://user-images.githubusercontent.com/112482781/188262439-c57fd4e2-e445-4a1b-a4f0-abf59f00f641.jpg)

**Comment by ehsandeep** [comment, marked as answer]
@adam9901 You can use `-nc` flag to disable colors being written to output file.

**Comment by adam9901** [comment]
Thank you, it works now

**Comment by prakhar0x01** [comment]
yes , command : nuclei -nc  -u https://subdomain.target.com/
